pkg/github/github_test.go

```diff
@@ -184,3 +166,210 @@ func TestGetAvailableReleasesUnmarshalingProblem(t *testing.T) {
 		"json: cannot unmarshal object into Go value of type []github.ReleaseInfo",
 	)
 }
+func TestCollectComponents(t *testing.T) {
```

Comment:
So I think this test here isn't really in the spirit of what the original issue was trying to look for. The test here is an API integration test because you are relying on the real API endpoint to return you data and as such its more fragile and also uses up our API quota. At the very least, this test should move into a separate file (to delineate it from real unit tests here; maybe `github_integration_test.go`) and also needs to have the [`testing.short()`](https://golang.org/pkg/testing/#hdr-Skipping) check at the start to not run when `go test -short` is used.

Reply:
Deferred to #140